### DISCUSSION TITLE
How can I get MNIST dataset?

### DISCUSSION THREAD

**Discussion by Hongseok-kim** [microsoft/AI-For-Beginners#309] [Q&A]
I've been executing perceptron notebook in my local environment. I don't know how to get MNIST dataset mentioned in this cell.

![Screenshot 2024-03-29 230000](https://github.com/microsoft/AI-For-Beginners/assets/7212600/c780be9c-edf1-49a4-a48a-413236f7a12a)

**Comment by jimsnns** [comment]
Did you try to uncomment the !wget command and run it? If you run your code on Jupyter Notebooks you just have to run the !wget command with the url as it is to download the dataset in your local machine.

If this is a regular.py file you need to use 'subprocess' module to download it with wget.

`import subprocess`

**Comment by devil0019** [comment]
Download the mnist dataset in kaggel and use it

Eswar Anne
